test/integration/integration_stream_decoder.cc

```diff
@@ -69,11 +69,17 @@ AssertionResult IntegrationStreamDecoder::waitForEndStream(std::chrono::millisec
   return AssertionSuccess();
 }
 
-void IntegrationStreamDecoder::waitForReset() {
+AssertionResult IntegrationStreamDecoder::waitForReset(std::chrono::milliseconds timeout) {
```

Comment:
must use result?

Reply:
Did you mean `ABSL_MUST_USE_RESULT`?

Reply:
Yep, seems like we want callers to check this?

Reply:
Ahh, maybe I can on this one - I couldn't on the other one I ported without changing signature, updating envoy filter examples, and then adding MUST_USE but I think Enovy filter examples doens't use this function